File: Servidor/AtHomeTFGCarlosHernandez/src/main/java/org/tfg/athometfgcarloshernandez/spring/model/response/CargarProductosResponseDTO.java
package org.tfg.athometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.athometfgcarloshernandez.spring.model.CajonDTO;
import org.tfg.athometfgcarloshernandez.spring.model.MuebleDTO;
import org.tfg.athometfgcarloshernandez.spring.model.ProductoDTO;

import java.util.List;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class CargarProductosResponseDTO {
    private List<ProductoDTO> productos;
    private List<CajonDTO> cajones;
    private List<MuebleDTO> muebles;
    private int idPropietario;
}

File: Cliente/AtHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/athometfgandroidcarloshernandez/data/model/response/CargarProductosResponseDTO.kt
package com.example.athometfgandroidcarloshernandez.data.model.response

import com.example.athometfgandroidcarloshernandez.data.model.CajonDTO
import com.example.athometfgandroidcarloshernandez.data.model.MuebleDTO
import com.example.athometfgandroidcarloshernandez.data.model.ProductoDTO

data class CargarProductosResponseDTO (
    val productos: List<ProductoDTO> = emptyList(),
    val cajones: List<CajonDTO> = emptyList(),
    val muebles: List<MuebleDTO> = emptyList(),
    val idPropietario: String = ""
)